packages/path-provider/example/lib/main.dart

```diff
@@ -105,6 +113,24 @@ class _MyHomePageState extends State<MyHomePage> {
               child: new FutureBuilder<Directory>(
                   future: _appDocumentsDirectory, builder: _buildDirectory),
             ),
+            new Cloumn(
+              children : <Widget>[
+                new Padding(
+                  padding: const EdgeInserts.add(16.0),
+                  child: new RaisedButton(
+                    child: const Test('${Platform.operatingSystem != "ios" ?
```

Comment:
const Test -> New Text 

You cannot have const here as the expression is not constant.